client/src/components/icons/FavoriteIcon.tsx

```diff
@@ -0,0 +1,56 @@
+import {StyleSheet, TouchableOpacity, View} from "react-native";
+import {MaterialCommunityIcons} from "@expo/vector-icons";
+import {updateFavorites} from "../../services/favorites.servise";
+import {useAppDispatch, useAppSelector} from "../../store/hooks";
+import {selectFavorites} from "../../store/favorites/favorites.selectors";
+import {addFavoriteThunk} from "../../store/favorites/favorites.slice";
+
+interface Props {
+    itemId: string
+}
+
+export const IconsStyles = StyleSheet.create({
+    iconCircleWrapper: {
+        position: "relative",
+        width: 32,
+        height: 32,
+        borderRadius: 50,
+        overflow: "hidden",
+    },
+    iconCirclePaper: {
+        position: "absolute",
+        top: "-35%",
+        left: "-35%",
+        width: "170%",
+        height: "170%",
+        backgroundColor: "white",
```

Comment:
Add `zIndex: -1` to paper in order to make in work in computer/android as well (as I sent to Mor, the icon was invisible but still clickable).

Reply:
done